src/number_theory/padics/padic_norm.lean

```diff
@@ -469,6 +469,13 @@ begin
     rwa nat.coprime_primes hp.1 hq.1, },
 end
 
+lemma valuation_prime_pow_eq_pow {p y : ℕ} [fact p.prime] : y = padic_val_nat p (p ^ y) :=
```

Comment:
This lemma looks to me like it would be better in reverse, and with `p` and `y` explicit
```suggestion
lemma valuation_prime_pow_eq_pow (p y : ℕ) [fact p.prime] : padic_val_nat p (p ^ y) = y :=
```